content/vault/v1.21.x/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -523,13 +523,15 @@ the secret to list, such as `secret`. This is specified as part of the URL.
 - `path` `(string: <required>)` – Specifies the path of the secrets to list.
   This is specified as part of the URL.
 
+- `exclude_deleted` `(bool: false)` – If set to true, only non-deleted entries will be returned.
+
 ### Sample request
 
 ```shell-session
 $ curl \
     --header "X-Vault-Token: ..." \
     --request LIST \
-    https://127.0.0.1:8200/v1/secret/metadata/my-secret
+    https://127.0.0.1:8200/v1/secret/metadata/my-secret?exclude_deleted=false
```

Comment:
```suggestion
    https://${VAULT_ADDR}/v1/secret/metadata/my-secret?exclude_deleted=false
```
We're slowly converting example code to use the `VAULT_ADDR` environment variable instead of hardcoding a localhost address